### ISSUE TITLE
Inconsistent links

### ISSUE THREAD

**Issue by kastnerp** [CornellSystemsEngineering/SYSEN-5160#6]
Hi, 

Thanks for your submission. I noticed you posted links to two different repos in the README:

https://github.com/CornellSystemsEngineering/SYSEN-5160/blob/1da24fc18ce5a13b97a194203f4700dc7fef3994/README.md?plain=1#L7
 
Which one is the one marked for submission? Also, could you add instructions on how to upload a sample .csv file so I can take a closer look?

Best,

Patrick

**Comment by bellesjchen** [comment]
Hello Professor,
Our final repo is the google doc(
https://docs.google.com/document/d/1x1Y2nP7ZF6BjZqnmfv3iMOEAJCe7g9O-JJlvDwo7cjk/edit#
)
I posted my github link because the content of the readme is actually same
with this google doc, just without the screenshots in the google doc.
For the steps of csv upload, you can directly select the demand data input
in the navigation bar and upload the instance csv to try. The csv link is
https://github.com/bellesjchen/sysen5160-scm-tool/blob/main/DataCoSupplyChainDataset.csv

Regards,
Sijie

Patrick Kastner ***@***.***> 于2022年5月22日周日 16:44写道：

**Comment by kastnerp** [comment]
Hi Sijie, 

I don't see a link to the final repo in the google doc link you posted. 

Could you please adapt edit the correct link in line 7, specifically this: `` [App](https://share.streamlit.io/fcelya/sysen5160-scm-tool/main/main.py) / [Repo](https://github.com/bellesjchen/sysen5160-scm-tool) ``

Thank you,

Patrick

**Comment by bellesjchen** [comment]
Hi Professor,

I don't know if the email I sent you before made you misunderstood. I
posted my github link in your link. The readme in it is the final report of
our group. The content in the readme and this Google link (
https://docs.google.com/document/d/1x1Y2nP7ZF6BjZqnmfv3iMOEAJCe7g9O-JJlvDwo7cjk/edit)
is the same, I don't think the link I put in your github is wrong.

Regards,
Sijie

Patrick Kastner ***@***.***> 于2022年5月22日周日 17:31写道：

**Comment by bellesjchen** [comment]
Hi Professor,
This is my github readme screenshot. It is the plain text version of our
group's final report, which is exactly the same as the text in the google
doc. The only difference is that there are some screenshots of our app in
the google doc.
[image: image.png]

Best,
Sijie

Patrick Kastner ***@***.***> 于2022年5月22日周日 17:31写道：

**Comment by fcelya** [comment]
Hi to both,

Both repos had the same working version of the application, but in order to be more consistent I have changed the Repo link so it points to the same one as the App link (bellesjchen/sysen5160-scm-tool currently is not uploaded to the Streamlit cloud so we pointed both to fcelya/sysen5160-scm-tool). The only difference between both repositories is the Readme, which we have changed in fcelya/sysen5160-scm-tool so it explains a little bit more about how to upload a custom .csv file. 

If the readme needs to hold the whole report we can add that.
